### PR TITLE
DOC-4279 : Add permissions parameter to space template configuration

### PR DESCRIPTION
**Pull request by wkhessairi**

### CODE REVIEW

File: docs/SpaceTemplates.rst [hunk 1]
```diff
@@ -108,11 +109,16 @@ In which:
...
 +-------------------------+------------------------------+----------------------------------------------------------------------------+
-| **homePageApplication** | ``SpaceApplication``         | The application to use in the home page of a space.                    |
+| **permissions**         | ``string``                   | The groups that have permissions to view and use this space template.      |
++-------------------------+------------------------------+----------------------------------------------------------------------------+
+| **homePageApplication** | ``SpaceApplication``         | The application to use in the home page of a space.                        |
 +-------------------------+------------------------------+----------------------------------------------------------------------------+
 | **applications**        | list of ``SpaceApplication`` | The list of the applications that are installed by default to a new space. |
 +-------------------------+------------------------------+----------------------------------------------------------------------------+
 
+.. tip:: The ``permissions`` parameter is a list of groups separated by ";" character.
```
Comment (plamarque) [inline docs/SpaceTemplates.rst]: member
```suggestion
.. tip:: The ``permissions`` parameter is a list of membership expressions separated by ";" character.
```

Comment (plamarque) [reply docs/SpaceTemplates.rst]: Does it accept simple group expressions without ":" ? such as "/platform/users"

**Review by plamarque — COMMENTED** [review]
permissions syntax may need more explanation. see my comment

File: docs/SpaceTemplates.rst (hunk 1 above)
Comment (wkhessairi) [reply docs/SpaceTemplates.rst]: No, it must fit the format MEMBERSHIP:GROUP_ID

**Review by plamarque — APPROVED** [review]

**Pull request merged**